src/coreclr/vm/interpexec.cpp

```diff
@@ -114,6 +137,9 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
             // keep it for such purposes until we don't need it anymore.
             pFrame->ip = (int32_t*)ip;
 
+
+            int offset = (int)(ip - (pFrame->startIp + sizeof(InterpMethod*) / sizeof(int32_t)));
```

Comment:
this looks like leftover ?

Reply:
Ah, yes, I had some debug print with the offset and forgotten to remove this.